compiler/x/amd64/codegen/OMRTreeEvaluator.cpp

```diff
@@ -126,6 +126,11 @@ TR::Register *OMR::X86::AMD64::TreeEvaluator::lloadEvaluator(TR::Node *node, TR:
    return reg;
    }
 
+TR::Register *OMR::X86::AMD64::TreeEvaluator::lReadBarrierEvaluator(TR::Node *node, TR::CodeGenerator *cg)
```

Comment:
This evaluator may go into common x86 code, it is same as the 32-bit one.

Reply:
changed according to the comments